sharkFinApi/sharkFinApi/Controllers/UserController.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace sharkFinApi.Controllers
+{
+    public class UserController : Controller
```

Comment:
Should replace this MVC controller class with an API controller.

Reply:
Looks like Matt M has already created a UsersController with some routes for the API.